charts/andi/templates/deployment.yaml

```diff
@@ -158,6 +158,8 @@ spec:
             value: '{{  .Values.footer.leftSideLink }}'
```

Comment:
You'll need to bump the versions of both of these charts, then bump the urbanos chart version and run `helm dep up` within the urbanos folder